speak.md

```diff
@@ -5,22 +5,99 @@ permalink: /speak/
 
-The sort of things we're looking for: 
+[Submit now!](https://www.papercall.io/pyconau2018)
+
+If you've never presented at a conference before and think you might like to try it, [submit now!](https://www.papercall.io/pyconau2018)
+
+We're looking for 30 minute talks on everything Python, open source, and tech-related.
+
+We welcome submissions from everybody, including those:
+
+* who have never given a conference talk before,
+* who are new to Python,
+* who have built interesting things in Python, 
+* who have broken interesting things with Python,
+* who have helped build Python, and
+* who have expertise to share with our community from fields outside of Python (and outside of technology).
+
+Thinking of something cool? We want to hear from you!
+
+You can submit a proposal for the main conference, and all of our four Specialist Tracks, [through PaperCall](https://www.papercall.io/pyconau2018) or reach us any time with questions at **<mailto:program@pycon-au.org>**.
+
+## Specialist Tracks
+
+The first day of PyCon AU is dedicated to specialist tracks. These tracks, also known as 'mini-confs', are curated by smaller specialist groups within Python community, and allow for more in-depth talks on the subject. 
+
+In 2018, these tracks will be: DjangoCon, Security and Privacy, Education, and Internet of Things. [Read more](http://2018.pycon-au.org/news/specialist-tracks).
+
+You can submit to any or all of these Specialist Tracks [through PaperCall](https://www.papercall.io/pyconau2018).
+
+(Applications for the Education Seminar Student Track will be taken here: https://docs.google.com/forms/d/e/1FAIpQLSeT8witEHPrvvYLr4QflguAYVr3zdvHKNdDnoFm2qR_HHDhBg/viewform).
+
+## Mentors
+
+In order to support speakers, we offer mentorship and feedback to those who would like it.
+
+A mentor is an experienced presenter who has volunteered to help other presenters. For first-time presenters, non-native English speakers, under-confident or uncertain speakers, or anyone who would just appreciate another set of eyes, our mentors are here to help. You’ll get the best results by forming a relationship with one mentor, rather than contacting several.
+
+Our mentors are happy to help with any of the following:
+
+* Exploring and brainstorming your interests to help you identify hidden things that would make great talks
+* Connecting you with experienced speakers to help build your proposal and talk
+* Reviewing your outline, slide deck, or presenter notes
+* Connecting you with rehearsal audiences or even just watching you present over a video conference as practice
+* Anything else that'd help you be at ease and excited about bringing your ideas to our audience!
+
+### Our mentors
+
+If you would like some help from a mentor, you can reach out to them directly:
+
+* [Katie Bell](): experienced international speaker, PyCon AU 2018 organiser, developer, and code archaeologist
```

Comment:
The links behind Katie's and Richard's names are missing.

Reply:
Pending preferred contact emails for both